### DISCUSSION TITLE
Expression Pedals are wrongly labelled 1-8 in app UI

### DISCUSSION THREAD

**Discussion by simonaglover95** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#78] [Editor App Feature Requests] [answered]
Smart Messages for expression messages ask you to select from Expression Pedals 1-8 instead of 1A, 1B, 2A, 2B. (only 4)

**Comment by konnsim** [comment, marked as answer]
Fixed, the exp pedal selections are now generated automatically based on the device descriptor json and labelled correctly.
